### PR TITLE
HBW-208 Add task type in task overview and make it sticky

### PR DESCRIPTION
**Pull request by vladdoncheg**

### CODE REVIEW

**Comment by vladdoncheg** [comment]
Скриншот 2019-11-12 15 34 18

**Comment by vladdoncheg** [comment]
Nov-12-2019 15-36-45

File: hbw/app/assets/stylesheets/hbw.sass [hunk 1]
```diff
@@ -451,15 +453,20 @@
...
       text-decoration: none
-
-    .title
       color: $claiming-purple
       margin-bottom: 15px
+
       &:hover
         color: $claiming-purple-hover
 
+
```
Comment (TimAle) [inline hbw/app/assets/stylesheets/hbw.sass]: extra line

File: hbw/app/assets/stylesheets/hbw.sass [hunk 2]
```diff
@@ -451,15 +453,20 @@
...
+    .title
+      color: #4A4A4A
```
Comment (TimAle) [inline hbw/app/assets/stylesheets/hbw.sass]: If this color used two or more times use variable.

File: hbw/app/javascript/packs/hbw/components/claiming/task_overview.jsx [hunk 3]
```diff
@@ -24,11 +24,12 @@ const HBWClaimingTaskOverview = ({
       <div className="bold header">
         {t('components.claiming.overview.header')}
       </div>
+      <div className="title">
```
Comment (TimAle) [inline hbw/app/javascript/packs/hbw/components/claiming/task_overview.jsx]: Tests!

File: hbw/app/javascript/packs/hbw/components/claiming/task_overview.jsx [hunk 4]
```diff
@@ -80,13 +81,14 @@ HBWClaimingTaskOverview.propTypes = {
...
-    due:         PropTypes.string,
-    description: PropTypes.string,
+    id:           PropTypes.string.isRequired,
+    priority:     PropTypes.number.isRequired,
+    name:         PropTypes.string.isRequired,
+    created:      PropTypes.string.isRequired,
+    icon:         PropTypes.string,
+    due:          PropTypes.string,
+    description:  PropTypes.string,
+    process_name: PropTypes.string,
```
Comment (TimAle) [inline hbw/app/javascript/packs/hbw/components/claiming/task_overview.jsx]: I think this one is required.

**Pull request merged**